authentication/pipeline/invite.py

```diff
@@ -0,0 +1,41 @@
+"""Pipeline functions for channel invitations"""
+from django.db import transaction
+
+from channels.api import get_admin_api
+from channels.models import ChannelInvitation
+
+
+def resolve_outstanding_channel_invites(
+    strategy, backend, *args, user=None, is_new=False, **kwargs
+):  # pylint: disable=unused-argument
+    """
+    Resolves outstanding channel invitations when a user joins
+
+    Args:
+        strategy (social_django.strategy.DjangoStrategy): the strategy used to authenticate
+        backend (social_core.backends.base.BaseAuth): the backend being used to authenticate
+        user (User): the current user
+        is_new (bool): True if the user just got created
+    """
+    if not is_new:
+        return {}
+
+    admin_api = get_admin_api()
+
+    # resolve all channel invitations by adding the user as a contributor and a subscriber
+    for invite_id in ChannelInvitation.objects.filter(
```

Comment:
Looking at coverage, it seems like this function is being run but it's not finding any items which match in the for loop and so the inner lines are never executed. Can you look into this?